<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>489</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>489</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidgetU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="fileSelectLabel">
       <property name="text">
        <string>File da analizzare</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="fileSelectLocation">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fileSelectBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidgetU2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>471</width>
      <height>187</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="checkInfo">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>In attesa...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QProgressBar" name="checkingBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="abortButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/abort</normaloff>:/icons/abort</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidgetU3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>471</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startCheck">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Analizza</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidgetU4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>471</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="comparationCheck">
       <property name="text">
        <string>Compara con</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="comparationString">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>471</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="checkTypeLabel">
       <property name="text">
        <string>Tipo di analisi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioMd5">
       <property name="text">
        <string>MD5</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSHA1">
       <property name="text">
        <string>SHA-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSHA256">
       <property name="text">
        <string>SHA-256</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_OpenCompare"/>
    <addaction name="action_Check"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuD_25">
    <property name="title">
     <string>&amp;D-25</string>
    </property>
    <addaction name="action_D25Website"/>
    <addaction name="action_D25FacebookPage"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Github"/>
    <addaction name="action_Sourceforge"/>
    <addaction name="separator"/>
    <addaction name="action_Statistics"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuD_25"/>
   <addaction name="menu"/>
  </widget>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Impostazioni...</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>Seleziona &amp;File...</string>
   </property>
  </action>
  <action name="action_Check">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Analizza</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Esci</string>
   </property>
  </action>
  <action name="action_D25Website">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/d-25.net</normaloff>:/icons/d-25.net</iconset>
   </property>
   <property name="text">
    <string>Il nostro &amp;sito!</string>
   </property>
   <property name="toolTip">
    <string>Il nostro sito!</string>
   </property>
  </action>
  <action name="action_D25FacebookPage">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/facebook</normaloff>:/icons/facebook</iconset>
   </property>
   <property name="text">
    <string>Su &amp;Facebook!</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;Informazioni su...</string>
   </property>
   <property name="toolTip">
    <string>Informazioni su</string>
   </property>
  </action>
  <action name="action_Credits">
   <property name="text">
    <string>&amp;Crediti</string>
   </property>
  </action>
  <action name="action_CheckUpdate">
   <property name="text">
    <string>Controlla &amp;Aggiornamenti</string>
   </property>
  </action>
  <action name="action_Github">
   <property name="text">
    <string notr="true">GITHUB_MENU</string>
   </property>
  </action>
  <action name="action_Sourceforge">
   <property name="text">
    <string notr="true">SOURCEFORGE_MENU</string>
   </property>
  </action>
  <action name="action_OpenCompare">
   <property name="text">
    <string>Apri un File per la &amp;comparazione...</string>
   </property>
  </action>
  <action name="action_Statistics">
   <property name="text">
    <string>&amp;Statistiche</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
